lab_1_keywords_tfidf/main.py

```diff
@@ -17,7 +17,17 @@ def clean_and_tokenize(text: str) -> Optional[list[str]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not isinstance(text, str):
+        print(None)
+    punc = '''1234567890!()[]{};:'"\,<>./?@#$%^&*_~'''
+    for elements in text:
```

Comment:
consider iterating over symbols of punctuation instead

Reply:
by doing it the way you are doing it right now, you update the text as many times as there are symbols in it 
you could make it faster and better by updating the text as many times as there are symbols of punctuation, as number of punctuation symbols is lower than length of the whole text